stories/wizard/wizard.stories.js

```diff
@@ -0,0 +1,868 @@
+                        <div class="fd-wizard__step-wrapper">
+                            <a class="fd-wizard__step-container" tabindex="0" aria-label="Products" aria-disabled="true">
+                                <span class="fd-wizard__step-indicator sap-icon--product"></span>
+                                <div class="fd-wizard__label-container fd-wizard__label-container--optional">
+                                    <span class="fd-wizard__label">Products</span>
+                                    <span class="fd-wizard__optional-text">(Optional)</span>
+                                </div>
+                            </a>
+                            <span class="fd-wizard__connector"></span>
+                        </div>
+                    </li>
+                    <li class="fd-wizard__step fd-wizard__step--upcoming fd-wizard__step--stacked">
+                        <div class="fd-wizard__step-wrapper">
+                            <a class="fd-wizard__step-container" tabindex="0" aria-label="Additional Information" aria-disabled="true">
+                                <span class="fd-wizard__step-indicator sap-icon--course-program"></span>
+                                <div class="fd-wizard__label-container">
+                                    <span class="fd-wizard__label">Additional Information</span>
+                                </div>
+                            </a>
+                            <span class="fd-wizard__connector"></span>
+                        </div>
+                    </li>
+                </ul>
+            </nav>
+            <section class="fd-wizard__content fd-wizard__content--s" id="wizard-section-7" style="min-height: 300px;">
+                <img class="" src='http://lorempixel.com/220/220/nature/4/' role="presentation" aria-label="John Doe" />
+                <div class="fd-wizard__next-step fd-wizard__next-step--floating">
+                    <button class="fd-button fd-button--emphasized">Next Step</button>
+                </div>
+            </section>
+            <footer>
+                <div class="fd-bar fd-bar--page-s fd-bar--floating-footer">
+                    <div class="fd-bar__right">
+                        <div class="fd-bar__element">
+                            <button class="fd-button fd-button--transparent">Cancel</button>
+                        </div>
+                    </div>
+                </div>
+            </footer>
+        </div>
+    </div>
+</div>
+`;
+
+mobile.parameters = {
+    docs: {
+        iframeHeight: 950,
+        storyDescription: `In mobile mode use the <code class="docs-code">fd-bar--floating-footer</code> modifier class to achieve a floating footer. The Next Step button is placed after the content when possible. It can be placed above the floating footer by applying the <code class="docs-code">fd-wizard\\_\\_next-step--floating</code> modifier class.
```

Comment:
For mobile devices, add the <code class="docs-code">fd-bar--floating-footer</code> modifier class to display a floating footer. The _Next Step_ button is placed after the content whenever possible. However, it can be placed above the floating footer by adding the <code class="docs-code">fd-wizard\\_\\_next-step--floating</code> modifier class.